app/models/api/measure.rb

```diff
@@ -13,19 +13,14 @@ class Measure < Api::Base
     meta_attribute :duty_calculator, :source
 
     attributes :id,
-               :additional_code,
```

Comment:
These were creating getters that are handled by the defined relationships in the DSL for an api resource